src/logger.c

```diff
@@ -11,7 +11,7 @@
 LoggerData g_logger;
 
 #ifndef min
-#define	min(a,b)	(((a) < (b)) ? (a) : (b))
+#define min(a,b) (((a) < (b)) ? (a) : (b))
```

Comment:
It's dangerous to put in a header because it depends on #include ordering. If it gets beyond 2 files I'll try to come up with a different plan.